src/main/java/com/revature/services/UserService.java

```diff
@@ -28,5 +25,7 @@
 	
 	public User login(String name, String password);
 	
+	public User validateUser(String username);
+	
```

Comment:
ValidateUser() method is to check whether the username already present in database.